x11-misc/light-locker/light-locker-1.8.0-r1.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools gnome2-utils
+
+DESCRIPTION="A simple locker using lightdm"
+HOMEPAGE="https://github.com/the-cavalry/light-locker"
+SRC_URI="${HOMEPAGE}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+consolekit +dpms +elogind gtk3 +screensaver -systemd +upower"
```

Comment:
Shouldn't only one be suggested for users? `+consolekit +elogind`

Reply:
This is not incompatible. elogind by default is just in the way to be pushed, so I think the consolekit default should be kept at least for now.

Reply:
tbh that line looks horrible. No default enabling any two session trackers at the same time please, in fact `+consolekit` looks wrong to begin with - `desktop` profile does that, and may switch in the future. Also, what's the deal with `-systemd` in here?

PS: I know these are coming from the earlier ebuild versions. As the maintainer of the package does not seem to frequent GitHub, it would be better to attach an ebuild.diff to the linked bug.